SECURITY.md

```diff
@@ -27,26 +40,24 @@ To streamline this process run the following:
 The pre-commit framework and detect secrets tooling requires Python
 
 ```bash
-# Initialize & activate a python virtual environment
-. ./python-virtual-env.sh
```

Comment:
this command doesnt work for me in my WSL console:-bash: ./python-virtual-env.sh: No such file or directory

Reply:
this is fine with me, i can just install pip. however, for some reason this kills the shell on the spot.

Reply:
Installing pre-commit hooks for first time setup...
pre-commit installed at .git/hooks/pre-commit
[INFO] Initializing environment for https://github.com/Yelp/detect-secrets.
[INFO] Initializing environment for https://github.com/Yelp/detect-secrets:pyahocorasick.
[INFO] Installing environment for https://github.com/Yelp/detect-secrets.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Detect secrets..........................................................

more error infor would be nice here